public/docs/ts/latest/guide/forms.jade

```diff
@@ -1,83 +1,90 @@
+  Any seasoned web developer can create an HTML form with all the right tags.
+  It's more challenging to create a cohesive data-entry experience that guides the
   user efficiently and effectively through the workflow behind the form.
 
-  *That* takes design skills that are, to be frank, well out of scope for this guide.
+  *That* takes design skills that are, to be frank, well out of scope for this page.
 
   It also takes framework support for
-  **two-way data binding, change tracking, validation, and error handling**
-  ... which we shall cover in this guide on Angular forms.
-
-  We will build a simple form from scratch, one step at a time. Along the way we'll learn how to:
-
-  - Build an Angular form with a component and template
-  - Use `ngModel` to create two-way data bindings for reading and writing input control values
-  - Track state changes and the validity of form controls
-  - Provide visual feedback using special CSS classes that track the state of the controls
-  - Display validation errors to users and enable/disable form controls
-  - Share information across HTML elements using template reference variables
-
-  Run the <live-example></live-example>.
+  *two-way data binding, change tracking, validation, and error handling*,
+  which is covered in this page on Angular forms.
+  <!-- CF: For the last two paragraphs, would the following rewrite work:
+   "Creating a cohesive data-entry experience requires design skills (which are out of
+   scope for this page), as well as framework support for
+   *two-way data binding, change tracking, validation, and error handling*,
+   which you'll learn about on this page."
+   -->
+
+  You'll build a simple form from scratch, one step at a time. Along the way you'll learn how to:
+
+  - Build an Angular form with a component and template.
+  - Use `ngModel` to create two-way data bindings for reading and writing input-control values.
+  - Track state changes and the validity of form controls.
+  - Provide visual feedback using special CSS classes that track the state of the controls.
+  - Display validation errors to users and enable/disable form controls.
+  - Share information across HTML elements using template reference variables.
+
+  You can run the <live-example></live-example> in Plunker and download the code from there.
 
 .l-main-section
 :marked
   ## Template-driven forms
 
-  Many of us will build forms by writing templates in the Angular [template syntax](./template-syntax.html) with
-  the form-specific directives and techniques described in this guide.
+  You can build forms by writing templates in the Angular [template syntax](./template-syntax.html) with
+  the form-specific directives and techniques described in this page.
 
 .l-sub-section
   :marked
-    That's not the only way to create a form but it's the way we'll cover in this guide.
+    That's not the only way to create a form but it's the way that's covered in this page.
+//    CF: Is this note necessary?
```

Comment:
It is but it should be more descriptive that there two choices: template-driven and reactive.